.ci/setup.sh

```diff
@@ -14,13 +14,13 @@ source "${cidir}/lib.sh"
 source /etc/os-release
 
 if [ "$ID" == fedora ];then
-	sudo -E dnf -y install automake yamllint coreutils moreutils bc make gcc
+	sudo -E dnf -y install automake yamllint coreutils moreutils bc make gcc hunspell pandoc
 elif [ "$ID" == centos ];then
 	sudo -E yum -y install epel-release
-	sudo -E yum -y install automake yamllint coreutils moreutils bc
+	sudo -E yum -y install automake yamllint coreutils moreutils bc hunspell pandoc
 elif [ "$ID" == ubuntu ];then
 	sudo apt-get -qq update
-	sudo apt-get install -y -qq make automake qemu-utils python-pip coreutils moreutils bc
+	sudo apt-get install -y -qq make automake qemu-utils python-pip coreutils moreutils bc hunspell pandoc
```

Comment:
@jodh-intel I added this commit to avoid Travis erroring out with:
```
ERROR: Need hunspell command
ERROR: spell check failed for document README.md
```
Is this OK to do?

Reply:
"yes", although a much better method is to simplify `.ci/setup.sh` so that it matches:

- https://github.com/kata-containers/documentation/blob/master/.ci/setup.sh

That way, the Travis config will end up calling https://github.com/kata-containers/tests/blob/master/.ci/setup_env_ubuntu.sh automatically, and that will install `hunspell` *et al*.

Reply:
Thanks! I'll updated it in that way.

Reply:
Does that require a specific ubuntu version? Cause it's failing to find `hunspell-en-gb`  and `yamllint` on trustry.

Reply:
... I take that back - we've *hard-coded* an unsupported version of Ubuntu! :sob: - so yeah, just update `dist:`.

Reply:
Nevermind, finally bionic is available on Travis :tada:, so I switched to it and the problem is solved.